cmd/virt-launcher/BUILD.bazel

```diff
@@ -113,7 +114,7 @@ group_file(
 passwd_entry(
     name = "qemu-user",
     gid = 107,
-    home = "",
+    home = "/",
```

Comment:
As I understand in current implementation the configs are expected to be in `os.Chown("/var/lib/swtpm-localca", 107, 107)`. Also I guess `qemu` user will not have the perms to write in `/`. Just wonder what the home dir is needed for in such case.

Reply:
Probably best to create a proper home for qemu, since I assume it's needed for `swtpm-create-user-config-files` anyway.

The original reason for this change was that some auth components (not sure which), deny the existence of a system user if it doesn't have a home. For example, try `su qemu` in a compute container.
`/` is usually the no-home default, and the fact that qemu couldn't write to it was actually a good thing!

Reply:
> For example, try su qemu in a compute container.

Afaik `su qemu` does not work because the user does not have a login shell defined. This can be solved  via `-s <shell>` flag:

```
84a89f842816:/ # su qemu 
This account is currently not available.
84a89f842816:/ # su qemu -s /bin/bash
qemu@84a89f842816:/> whoami
qemu
qemu@84a89f842816:/>
```

Reply:
Without any home dir, not even `/`?
I get:
```
$ kubectl exec -it virt-launcher-vmi-fedora-7vkc7 -c compute -- /bin/bash
bash-4.4# cat /etc/passwd 
qemu:x:107:107:user::/bin/bash
root:x:0:0:root:/root:/bin/bash
bash-4.4# su qemu
su: user qemu does not exist
bash-4.4# sed -i 's|::|:/:|' /etc/passwd
bash-4.4# su qemu
bash-4.4$ 
```

Reply:
:thinking: hm, maybe you are right. I tried in a custom container